docs/source/ml.rst

```diff
@@ -130,13 +130,8 @@ the pre-defined ones. Create the object and pass it as the ``model`` parameter
 of the :class:`~civis.ml.ModelPipeline`. Your model must follow the
 scikit-learn API, and you will need to include any dependencies as
 :ref:`custom-dependencies` if they are not already installed in
-CivisML. Preinstalled libraries available for your use include:
-
-- `scikit-learn <http://scikit-learn.org>`_ v0.19.1
-- `glmnet <https://github.com/civisanalytics/python-glmnet>`_ v2.0.0
-- `xgboost <http://xgboost.readthedocs.io>`_ v0.6a2
-- `muffnn <https://github.com/civisanalytics/muffnn>`_ v1.2.0
-- `civisml-extensions <https://github.com/civisanalytics/civisml-extensions>`_ v.0.1.6
+CivisML. Please check `here <https://civis.zendesk.com/hc/en-us/articles/360000260011-CivisML>`_
+for the available pre-installed libraries and their versions.
```

Comment:
Comment: This does have the downside of requiring that people have Platform access before they see what libraries we have available. It seems worth it to avoid the dependency between CivisML updates and API client updates, though.